source/visualisation.rst

```diff
@@ -2,146 +2,279 @@
 
 Manipulating the View
 .....................
 
-We can manipulate the view with mouse actions, clicking and dragging with the mouse will transform the view.  The following table describes the mouse button to the transformation applied.
+We can manipulate the view with mouse actions: clicking and dragging with the mouse in the graphics window area allows you to rotate, pan and zoom the view. The following table describes which mouse button controls which transformation.
 
 ============ ==============
 Mouse Button Transformation
 ============ ==============
-Left         Tumble
+Left         Tumble/Rotate
 ------------ --------------
 Middle       Pan/Translate
 ------------ --------------
-Right        fly-zoom
+Right        Fly Zoom
 ------------ --------------
-Shift+Right  Camera zoom
+Shift+Right  Camera Zoom
 ============ ==============
 
-When we transform the view with the mouse you can see from the view pane the change in the camera controls (see :numref:`dtp_cp_vis_viewpane`).
+When we transform the view with the mouse you can see the corresponding settings change in SimpleViz' view page (see :numref:`fig_dtp_cp_vis_simpleviz_viewpage`). You can also directly enter values into the controls. Regular Fly Zoom moves the eye point closer to the lookat point. Camera Zoom changes the angle of view but the eye doesn't move; if you make a very wide angle of view and then move in close, it is like looking through a very wide angle lens. The Tumble/Rotate control rotates about an axis in the scene, like pulling on a tangent to a large sphere filling the window. Play with these controls until they make sense to you. If things start looking too weird, click the 'View All' button to restore a normal view.
+
+In real life you can see from in front of your eyes to infinity, albeit not all in focus. In typical 3-D computer graphics everything is in focus, but you can only see a range of distances in front of your eye in the direction of the 'lookat point': between the near and far clipping plane distances. When you view the world in perspective mode (the default in SimpleViz), the part of space you see is called a *viewing frustum*, which is a pyramid seen from above but with its top chopped off at the near clipping plane. By turning off perspective you get a *parallel projection* where sizes of objects are unchanged by distance from the eye, like an extreme telephoto lens effect. Note that parallel projection uses the near and far clipping planes in exactly the same way.
```

Comment:
would be good to include the diagram from @hsorby's slides illustrating these concepts - if that is possible?